servers/republik/modules/crowdfundings/graphql/schema-types.js

```diff
@@ -260,6 +260,32 @@ type PaymentSource {
   expYear: Int!
 }
 
+enum CancellationCategoryType {
```

Comment:
What went away now is also the "system" cancellation, due to a dormant abo et al. Did like that before. So we'd understand why a `membership.active` flag is set to `false`.

System cancellations could be:
- Viable, dormant membership recognized, cancelling this membership
- Payment did not arrive within expected time period